### ISSUE TITLE
Crashing while reading .cbz files on Kobo Sage (memory allocation issue?)

### ISSUE THREAD

**Issue by Estebiu** [koreader/koreader#11499]
KOReader version: 2024.1
Device: Kobo Sage

#### Issue

Crashes.

#### Steps to reproduce
Sometimes when reading a .cbz KoReader crashes. I do not know what causes it; but I think it's because the Kobo Sage has only 512mb of ram and sometimes that isn't enough (malloc error)?

In this istance, I'm reading Witch Hat Atelier. As you can see from the log, Koreader crashes at least 2 times each volume, which is quite frustrating. After restarting koreader I can go past the pages the pages where it crashed, thought it'll just crash again soon. 

This issue still presented itself sometimes with other mangas extremely rarely; although this amount of crashes is unforeseen.

##### `crash.log` 
[kobo_crash.log](https://github.com/koreader/koreader/files/14425606/kobo_crash.log)

**Comment by Frenzie** [comment]
Indeed, that does seem to be the reason. They do seem to be fairly large images; it might be an idea to resize them slightly.

@NiLuJe preprocesses them to exactly the right size:
https://www.mobileread.com/forums/showpost.php?p=3728291&postcount=17

**Comment by axu2** [comment]
Maybe preprocess the manga with this?

- https://github.com/ciromattia/kcc

**Comment by mergen3107** [comment]
User guide's manga section has it all! Please have a look, and let me know if you have questions- I wrote that section
